src/commands/pubsub-shardnumsub.json

```diff
@@ -20,6 +20,25 @@
                 "multiple": true
             }
         ],
+        "key_specs": [
+            {
+                "flags": [
+                    "NOT_KEY"
+                ],
+                "begin_search": {
+                    "index": {
+                        "pos": 1
+                    }
+                },
+                "find_keys": {
+                    "range": {
+                        "lastkey": -1,
+                        "step": 1,
+                        "limit": 0
+                    }
+                }
+            }
+        ],
```

Comment:
Also, with this change, doesn't it update commands.def ?

Reply:
Yes it does, should I push those changes as well?

Reply:
Yes please

Reply:
It needs to be committed to make the CI pass.

Reply:
I have pushed the commands.def file. I'll push the test cases I have written once I have finished running the tests.